include/sys/zfs_context.h

```diff
@@ -633,6 +655,9 @@ extern void delay(clock_t ticks);
 #define	max_ncpus	64
 #define	boot_ncpus	(sysconf(_SC_NPROCESSORS_ONLN))
 
+#define	max(a, b)	(((a) > (b)) ? (a) : (b))
```

Comment:
Those macros were added because the Illumos kernel has them and I was trying to avoid changing the ported code from there as much as possible. A quick grep shows that the ICP uses `min()` twice and never actually uses `max()`, so I can just change these to the existing one.